content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1423,6 +1423,19 @@ Used on: Ingress
 Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassName`.
 {{< /note >}}
 
+### kubernetes.io/cluster-service (deprecated) {#kubernetes-io-cluster-service}
+
+Type: Label
+
+Example: kubernetes.io/cluster-service: "true"
```

Comment:
_Minor non-blocking nit_: Adjust the formatting of the example block to match the style of all the other examples on this [page](https://kubernetes.io/docs/reference/labels-annotations-taints/).

```suggestion
Example: `kubernetes.io/cluster-service: "true"`
```

Reply:
@dipesh-rawat incorporated suggestion.